style.css

```diff
@@ -0,0 +1,61 @@
+body {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    font-size: 1.6rem;
+    line-height: 1.5;
+    text-align: center;
+    color: #333;
+    margin: 0;
+  }
+
+  #main {
+    margin: 40px 10px;
+    padding: 25px;
+    border-radius: 5px;
+    background-color: crimson;
+  }
+
+img {
+    max-width: 100%;
+    display: block;
+    height: auto;
```

Comment:
Please follow CSS best practices. Please read [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more info